gateway/proxy_muxer.go

```diff
@@ -362,7 +363,12 @@ func (m *proxyMux) swap(new *proxyMux) {
 			}
 		}
 	}
-
+	p := m.getProxy(config.Global().ListenPort)
+	if p != nil && p.router != nil {
+		// All APIs processed, now we can healthcheck
+		// Add a root message to check all is OK
+		p.router.HandleFunc("/"+config.Global().HealthCheckEndpointName, liveCheckHandler)
```

Comment:
Why are we doing this twice?

We are already registering this here https://github.com/TykTechnologies/tyk/blob/8650e1803309405da9ecd814da5e8efbf43732d5/gateway/server.go#L416

Reply:
thanks, I made the change